controllers/admin/AdminImportController.php

```diff
@@ -1428,32 +1992,34 @@ protected function categoryImportOne($info, $default_language_id, $id_lang, $for
         //copying images of categories
         if (isset($category->image) && !empty($category->image)) {
             if (!(AdminImportController::copyImg($category->id, null, $category->image, 'categories', !$regenerate))) {
-                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.', array(), 'Admin.Advparameters.Notification');
+                $this->warnings[] = $category->image.' '.$this->trans('cannot be copied.',
+                    array(),
+                    'Admin.Advparameters.Notification'
+                );
             }
         }
         // If both failed, mysql error
         if (!$res) {
-            $this->errors[] = $this->trans(
-                '%1$s (ID: %2$s) cannot be %3$s',
-                array(
-                    (isset($info['name']) && !empty($info['name']))? Tools::safeOutput($info['name']) : 'No Name',
-                    (isset($info['id']) && !empty($info['id']))? Tools::safeOutput($info['id']) : 'No ID',
-                    ($validateOnly?'validated':'saved')
-                ),
-                'Admin.Advparameters.Notification'
-            );
-            $error_tmp = ($field_error !== true ? $field_error : '').(isset($lang_field_error) && $lang_field_error !== true ? $lang_field_error : '').Db::getInstance()->getMsgError();
-            if ($error_tmp != '') {
-                $this->errors[] = $error_tmp;
+            $this->errors[] = $this->trans('%1$s (ID: %2$s) cannot be %3$s', array(
+                !empty($info['name']) ? Tools::safeOutput($info['name']) : 'No Name',
+                !empty($categoryId) ? Tools::safeOutput($categoryId) : 'No ID',
+                $validateOnly ? 'validated' : 'saved',
```

Comment:
Same comment about translation placeholder. Please use a variable name surrounded by `%%` an add it in the second parameter.